.vscode/launch.json

```diff
@@ -87,6 +87,34 @@
       "smartStep": true,
       "internalConsoleOptions": "openOnSessionStart",
       "outputCapture": "std"
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Workflow Example Server (Websocket)",
+      "program": "${workspaceFolder}/examples/workflow-server/bundle/wf-glsp-server-node.js",
+      "args": ["--webSocket", "--port", "8081"],
+      "env": {
+        "NODE_ENV": "development"
+      },
+      "sourceMaps": true,
+      "outFiles": [
+        "${workspaceRoot}/examples/workflow-server/bundle/**/*.js",
+        "${workspaceRoot}/node_modules/@eclipse-glsp/*/lib/**/*.js",
+        "${workspaceRoot}/examples/workflow-server/*/lib/**/*.js",
+        "${workspaceRoot}/packages/**/*/lib/**/*.js"
+      ],
+      "sourceMapPathOverrides": {
```

Comment:
Could you maybe add a short comment on why this is necessary and what the deal is with the `webpack` and `meteor` URIs?